source/admin_manual/list_of_events.rst

```diff
@@ -206,31 +206,49 @@ Common fields
-| local_name        | TLS SNI.                                                   |
-+-------------------+------------------------------------------------------------+
-| local_ip          | Local IP client connected to.                              |
-+-------------------+------------------------------------------------------------+
-| remote_ip         | Remote IP of client.                                       |
-+-------------------+------------------------------------------------------------+
-| local_port        | Local port client connected to.                            |
-+-------------------+------------------------------------------------------------+
-| remote_port       | Remote port of client.                                     |
-+-------------------+------------------------------------------------------------+
-| real_local_ip     | Real local IP as seen by the server.                       |
-+-------------------+------------------------------------------------------------+
-| real_remote_ip    | Real remote IP as seen by the server.                      |
-+-------------------+------------------------------------------------------------+
-| real_local_port   | Real local port as seen by the server.                     |
-+-------------------+------------------------------------------------------------+
-| real_remote_port  | Real remote port as seen by the server.                    |
-+-------------------+------------------------------------------------------------+
++--------------------+------------------------------------------------------------------------+
+| Field             | Description                                                             |
++===================+=========================================================================+
+| mechanism         | Name of used SASL mechanism (e.g. PLAIN).                               |
++-------------------+-------------------------------------------------------------------------+
+| service           | Service name, such as imap3, pop3, lmtp.                                |
++-------------------+-------------------------------------------------------------------------+
+| transport         | Transport security indicator (insecure, TLS, trusted).                  |
++-------------------+-------------------------------------------------------------------------+
+| session           | Session identifier.                                                     |
++-------------------+-------------------------------------------------------------------------+
+| certificate_user  | Username from certificate                                               |
++-------------------+-------------------------------------------------------------------------+
+| client_id         | Expands to client ID request as IMAP arglist.                           |
+|                   | Needs :dovecot_core:ref:`imap_id_retain=yes <setting-imap_id_retain>`.  |
```

Comment:
Looks like it doesn't give an error, but the link doesn't work. Remove the "setting-" here and below.